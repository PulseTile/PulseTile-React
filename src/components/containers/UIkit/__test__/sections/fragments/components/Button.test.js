import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Buttons from '../../../../sections/fragments/components/Buttons';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Buttons />', () => {
  it('should renders Buttons with props correctly', () => {
    const component = shallow(<Buttons />);
    expect(component).toMatchSnapshot();
    component.find('.btn.btn-success.btn-inverse.btn-dropdown-toggle').simulate('click');
    expect(component).toMatchSnapshot();
    component.find('.btn.btn-success.btn-inverse.btn-dropdown-toggle').simulate('click');
    expect(component).toMatchSnapshot();
  });
});
