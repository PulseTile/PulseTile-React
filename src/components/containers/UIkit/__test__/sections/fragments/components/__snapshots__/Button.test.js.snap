// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Buttons /> should renders Buttons with props correctly 1`] = `
<div
  className="ui-section"
  id="buttons"
>
  <strong
    className="ui-title"
  >
    Buttons
  </strong>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Button color variations
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons inverse
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-inverse"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons without border
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-inverse btn-none-border"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons disabled
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success disabled"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success disabled btn-inverse"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success disabled btn-inverse btn-none-border"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons with icons
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success"
        >
          <i
            className="btn-icon fa fa-filter"
          />
        </button>
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-filter"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons for dropdown menu
    </strong>
    <div
      className="wrap-control-group"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse btn-dropdown-toggle"
          onClick={[Function]}
          type="button"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons size
    </strong>
    <div
      className="wrap-control-group"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-sm"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-smaller"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-square"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
        <button
          className="btn btn-success btn-inverse btn-square  btn-sm"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
        <button
          className="btn btn-success btn-inverse btn-square  btn-smaller"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
      </div>
    </div>
    <div
      className="wrap-control-group"
    >
      <button
        className="btn btn-success btn-inverse btn-block"
      >
        <i
          className="btn-icon fa fa-check"
        />
         
        <span
          className="btn-text"
        >
          Button text
        </span>
      </button>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons rounded
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse btn-rounded"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-rounded btn-square"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons text weight
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-bold"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-icon-normal"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-icon-normal btn-bold"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Buttons /> should renders Buttons with props correctly 2`] = `
<div
  className="ui-section"
  id="buttons"
>
  <strong
    className="ui-title"
  >
    Buttons
  </strong>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Button color variations
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons inverse
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-inverse"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons without border
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-inverse btn-none-border"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons disabled
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success disabled"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success disabled btn-inverse"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success disabled btn-inverse btn-none-border"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons with icons
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success"
        >
          <i
            className="btn-icon fa fa-filter"
          />
        </button>
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-filter"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons for dropdown menu
    </strong>
    <div
      className="wrap-control-group"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse btn-dropdown-toggle"
          onClick={[Function]}
          type="button"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons size
    </strong>
    <div
      className="wrap-control-group"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-sm"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-smaller"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-square"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
        <button
          className="btn btn-success btn-inverse btn-square  btn-sm"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
        <button
          className="btn btn-success btn-inverse btn-square  btn-smaller"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
      </div>
    </div>
    <div
      className="wrap-control-group"
    >
      <button
        className="btn btn-success btn-inverse btn-block"
      >
        <i
          className="btn-icon fa fa-check"
        />
         
        <span
          className="btn-text"
        >
          Button text
        </span>
      </button>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons rounded
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse btn-rounded"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-rounded btn-square"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons text weight
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-bold"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-icon-normal"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-icon-normal btn-bold"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Buttons /> should renders Buttons with props correctly 3`] = `
<div
  className="ui-section"
  id="buttons"
>
  <strong
    className="ui-title"
  >
    Buttons
  </strong>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Button color variations
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons inverse
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-inverse"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons without border
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-inverse btn-none-border"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-default"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-primary"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-success"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-info"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-warning"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-inverse btn-none-border btn-danger"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons disabled
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success disabled"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success disabled btn-inverse"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success disabled btn-inverse btn-none-border"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons with icons
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success"
        >
          <i
            className="btn-icon fa fa-filter"
          />
        </button>
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-filter"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons for dropdown menu
    </strong>
    <div
      className="wrap-control-group"
    >
      <div
        className="control-group with-indent left open"
      >
        <button
          className="btn btn-success btn-inverse btn-dropdown-toggle"
          onClick={[Function]}
          type="button"
        >
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons size
    </strong>
    <div
      className="wrap-control-group"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-sm"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-smaller"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-square"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
        <button
          className="btn btn-success btn-inverse btn-square  btn-sm"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
        <button
          className="btn btn-success btn-inverse btn-square  btn-smaller"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
      </div>
    </div>
    <div
      className="wrap-control-group"
    >
      <button
        className="btn btn-success btn-inverse btn-block"
      >
        <i
          className="btn-icon fa fa-check"
        />
         
        <span
          className="btn-text"
        >
          Button text
        </span>
      </button>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons rounded
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse btn-rounded"
        >
          <i
            className="btn-icon fa fa-check"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-rounded btn-square"
        >
          <i
            className="btn-icon fa fa-check"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="ui-sub-section"
  >
    <strong
      className="ui-sub-title"
    >
      Buttons text weight
    </strong>
    <div
      className="wrap-control-group hide-indent-bottom"
    >
      <div
        className="control-group with-indent left"
      >
        <button
          className="btn btn-success btn-inverse"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-bold"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-icon-normal"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
        <button
          className="btn btn-success btn-inverse btn-icon-normal btn-bold"
        >
          <i
            className="btn-icon fa fa-table"
          />
           
          <span
            className="btn-text"
          >
            Button text
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
